src/test/groovy/com/datadog/ddwaf/WafTest.groovy

```diff
@@ -0,0 +1,162 @@
+/*
+ * Unless explicitly stated otherwise all files in this repository are licensed
+ * under the Apache-2.0 License.
+ *
+ * This product includes software developed at Datadog
+ * (https://www.datadoghq.com/). Copyright 2023 Datadog, Inc.
+ */
+
+package com.datadog.ddwaf
+
+import org.junit.Test
+import org.junit.experimental.runners.Enclosed
+import org.junit.runner.RunWith
+
+@RunWith(Enclosed)
+class WafTest implements WafTrait {
+
+    static class InitializationTest {
+        @Test
+        void testLibVersion() {
```

Comment:
We have this at:
```
    @Test
    void 'the version is correct'() {
        assert Waf.version =~ Waf.LIB_VERSION
    }
```

I don't think we need this (also not related to any affeced code in this PR).